### PR TITLE
Rend2

### PR DESCRIPTION
**Pull request by SomaZ**
So, this is a working version of a fork of the ioquake3 renderer2 for the Jedi Academy Multiplayer. This is still WIP, but all the essential vanilla renderer features are implemented.

Enabling the renderer via comandline:
"cl_renderer rd-rend2;vid_restart"

Pull request currently includes:
- rend2 renderer
- ~~fix for the github actions linux x86 build~~ (not needed anymore and removed)
- fixes for compiling the mp vanilla and dedicated renderers without gore
- correct sdl includes

Special thanks to following people:
@SmileTheory for the original renderer2
@eezstreet and @ensiform for their effort in porting the renderer to Jedi Academy
@xycaleth for writing nearly all the new essential parts of the renderer

### CODE REVIEW

File: .github/workflows/rend2.yml
Comment (Razish) [inline .github/workflows/rend2.yml:1]: Is this file actually necessary if we merge? We may just need to update the existing github actions to supply `-DBuildMPRend2=On`

Comment (SomaZ) [reply .github/workflows/rend2.yml:1]: Nah, can get rid of it again. Just needed it to create builds on my fork because its obviusly not the [master] branch I push to.

**Review by Razish — DISMISSED** [review]

**Review by AlexCS1337 — APPROVED** [review]
I tested it and works well, I think Somaz said something was missing from my screenshot but I don't think it was something major

**Comment by SomaZ** [comment]
> I tested it and works well, I think Somaz said something was missing from my screenshot but I don't think it was something major

I was thinking I did something stupid with the hdr ambient lighting from the lightgrid, but it turns out everything is fine from the code perspective. Just did something stupid in blender when baking the lightgrid.

**Review by Razish — APPROVED** [review]

**Review by taysta — APPROVED** [review]
Have been testing this thoroughly on SomaZ's map on my AMD 6900xt on Windows 10, no issues.
